src/main/resources/templates/home.html

```diff
@@ -0,0 +1,41 @@
+<html>
+<body>
+
+<form action="/login" method="post">
+    <input type="text" placeholder="Enter your name" name="userName"/>
+    <button type="submit">Login</button>
+</form>
+
+
+<form action="/add-recipe" method="post">
+    <input type="text" placeholder="Add Recipe" name="createRecipe"/>
+    <button type="submit">Add Recipe</button>
+</form>
+
+
+<ol>
+    {{#recipes}}
+    <li>
+        {{id}} {{text}}
+        <form action="/modify-recipe" method="post">
+            <input type="hidden" value="{{id}}" name="id"/>
+            <button type="submit">Edit</button>
+        </form>
+    </li>
+    {{/recipes}}
```

Comment:
fyi, I don't think you need 2 separate lists for Edit and Delete. You could just combine them to show an Edit form along side a Delete form for each recipe. But, this works too.